### DISCUSSION TITLE
Wrong OS or Just uncommon UA Pattern?

### DISCUSSION THREAD

**Discussion by nvysyaraju-sift** [nielsbasjes/yauaa#1033] [Q&A] [answered]
Hello @nielsbasjes 

Considering this as a UA string "Firefox 4.0", the parser successfully returns the Browser and version but it returns the same value for OS too. Not sure if this needs a fix or its better to consider this as an uncommon UA string pattern.

**Comment by nielsbasjes** [comment, marked as answer]
I consider this a very uncommon user agent and to me the current result is even better than what I would have expected.

**Reply by nvysyaraju-sift** [reply]
Thank you.
